packages/block-library/src/file/block.json

```diff
@@ -33,10 +33,17 @@
 			"type": "string",
 			"source": "html",
 			"selector": "a[download]"
+		},
+		"showInlineEmbed": {
+			"type": "boolean"
+		},
+		"embedHeight": {
+			"type": "number"
 		}
 	},
 	"supports": {
 		"anchor": true,
 		"align": true
-	}
+	},
+	"script": "utils.js"
```

Comment:
What is doing?

Reply:
It seems it's a way to tell pack to generate a separate script? Why not just a package because that's our way of doing this in Gutenberg? 

Alternatively, if we implement the `frontend_script` proposal, we could introduce a convention like this (I see it more as a file in the block  folder with the property here though). Having the property here will persist in the block registration... which I think is confusing.